src/inverseRobot.js

```diff
@@ -4,10 +4,19 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
+
+  for (const key in robot) {
+    if (obj.hasOwnProperty([robot[key]])) {
```

Comment:
```suggestion
    if (obj.hasOwnProperty(robot[key])) {
```

also move `robot[key]` in the variable